previous.html

```diff
@@ -12,6 +12,8 @@
 
 <h3>Version 5</h3>
 
+<p>All User Guides for <a href="http://downloads.openmicroscopy.org/help/archives/OMERO_User_Guides_5_2_5.zip">Version 5.2.5</a> (Zipped PDFs 139 MB)</p>
```

Comment:
When the new ImageJ chapter is swapped into the archive the size changes - I don't know why the 2 are different - some zip voodoo.